src/index.html

```diff
@@ -4,9 +4,41 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar__start-day--mon calendar__month-length--31">
```

Comment:
you can use more semantic tags, please use ul instead of divs

```
<ul class="calendar calendar__start-day--mon calendar__month-length--31">
  <li class="calendar__day"></li>
  <li class="calendar__day"></li>
  <li class="calendar__day"></li>
  etc...
</ul>
```